tags.go

```diff
@@ -49,6 +59,15 @@ type TagDeleteInput struct {
 	Id ID `json:"id"`
 }
 
+//#region Helpers
+
+func ValidateTagKey(key string) error {
```

Comment:
What do you think about making this a `func (t *Tag) ValidateTagKey()...` method?
The `TagKeyRegex` and `TagKeyErrorMsg` vars could live in there too.

Reply:
ou i like that idea!

Reply:
this was not possible - we paired on it together